docs/management_api.md

```diff
@@ -285,6 +285,127 @@ curl http://localhost:8081/models/noop/all
 ]
 ```
 
+`GET /models/{model_name}/{model_version}?customized=true`
```

Comment:
What do you think about changing this to to `customized_metadata=true`

Reply:
Original I thought of sth like "customized_metadata". I feel the parameter name is a bit long after second thoughts, so I chose "customized"